src/renderer/components/Handle.tsx

```diff
@@ -84,7 +84,7 @@ const HandleElement = memo(
     }
 );
 
-const Div = chakra('div', {
+export const Div = chakra('div', {
```

Comment:
This element is never used outside this file.
```suggestion
const Div = chakra('div', {
```